test/bin/mdx-test/expect/block-type/test-case.md

```diff
@@ -0,0 +1,24 @@
+It is possible to explicitly state the type of a block using the
+`block-type` label, working around language header and content based
+inference which can sometime lead to troublesome error messages.
```

Comment:
I'd say "it can sometimes fail and lead to confusing error messages" to state why these error messages happen (because I've been curious myself).